backend-api/template.yaml

```
@@ -773,6 +784,11 @@ Resources:
             Method: get
             RestApiId: !Ref SessionsApi
       Role: !GetAtt AsyncActiveSessionLambdaRole.Arn
+      Environment:
+        Variables:
+          KID_ARN: !GetAtt KMSEncryptionKey.Arn
```

```suggestion
          ENCRYPTION_KEY_ARN: !GetAtt KMSEncryptionKey.Arn
```

how do we feel about this?

---

Yes I like it, it's less ambiguous. I think we just wanted to ensure the ARN part was in there as we've seen before references to a KID when the value is actually an ARN but I think your suggestion is best of both

---

Done